app/assets/stylesheets/base.scss

```diff
@@ -9,12 +9,15 @@
 
 @import "exhibits/variables_exhibitions";
 @import "exhibits/variables_bootstrap";
-@import "exhibits/base";
-@import "exhibits/header";
+@import "exhibits/base.scss";
```

Comment:
No ".sccs" extension needed.

Reply:
Thanks! That's why one shouldn't push changes at 4pm on a Friday :)